osm_jet/fields.py

```diff
@@ -32,3 +45,134 @@ class DjangoManyToMany(models.ManyToManyField):
     def formfield(self, **kwargs):
         kwargs['widget'] = DjangoManyToManyWidget
         return super().formfield(**kwargs)
+
+
+class SortedManyToManyWidget(CheckboxSelectMultiple):
```

Comment:
I think it might be better to inherit from `SortedCheckboxSelectMultiple` to save the code duplication.

Reply:
All the functions have changed since we're using the new style of widget rendering. `SortedCheckboxSelectMultiple` has all the `get_selected_and_unselected()` code in it's `render()` function and it's `build_attrs()` function has different args

Reply:
OK, that is reasonable. :)